media-plugins/gst-plugins-dvb/gst-plugins-dvb-1.20.0.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+GST_ORG_MODULE=gst-plugins-bad
+
+inherit gstreamer-meson
+
+DESCRIPION="DVB device capture plugin for GStreamer"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~mips ~ppc ~ppc64 ~riscv ~x86"
+IUSE=""
+
+RDEPEND=""
```

Comment:
Ditto everywhere (and for blank IUSE):
```suggestion
```

Reply:
My style is to be explicit and actually write it out like was done originally here.

Reply:
@leio do you mean just keep these empty `RDEPEND=""` or would you prefer to keep empty `IUSE=""` ?

Reply:
I restored empty `RDEPEND=""` entries now.
Please let me know if you want empty `IUSE=""` back as well.

Reply:
I'm fine with USE="" not being there anymore